example/platform.html

```diff
@@ -0,0 +1,105 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Viewer</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <script src="../amp-viewer.js"></script>
```

Comment:
Probably ok for first commit, but give unpackaged/ES6 nuances won't work very soon. You'll need to have a sample binary built.